src/fides/api/oauth/system_manager_oauth_util.py

```diff
@@ -88,20 +88,22 @@ async def verify_oauth_client_for_system_from_request_body(
 
     Yields a 403 forbidden error if not.
     """
-    return has_system_permissions(
+
+    system = has_system_permissions(
         system_auth_data=system_auth_data,
         authorization=authorization,
         security_scopes=security_scopes,
         db=db,
     )
+    return system
 
 
 async def verify_oauth_client_for_system_from_fides_key(
     security_scopes: SecurityScopes,
     authorization: str = Security(oauth2_scheme),
     db: Session = Depends(get_db),
     system_auth_data: SystemAuthContainer = Depends(_get_system_from_fides_key),
-) -> str:
+) -> Union[str, System]:
```

Comment:
I believe this method always returns a string even though the underlying method is capable of returning multiple types, same for the method above which I believe always returns a system.

This type change makes it harder to tell what is actually returned here, I would prefer an ignore in this case.

(Granted, my implementation here was confusing.  When I did something similar in plus, it was implemented in a cleaner way).

Reply:
Ok, I'll take a look here. If it always returns a string, then it should be typed as such